general/server/media/shows.md

```diff
@@ -33,6 +37,13 @@ Shows
 > [!NOTE]
 > Season folders shouldn't contain the series name, otherwise Jellyfin can in certain cases (Stargate SG-1 due to the dash and one, for instance) misdetect your episodes and put them all under the same season.
 
+## Show Extras
+
+Show extras, sometimes called specials, can be added in the `Series 00` folder. In case your metadata provider does not provide information about the extra, it is recommended to use a name which describes the content of the special instead of naming it `Episode S00Exy.mkv`. This is done to avoid wrong metadata being pulled for the extra.
```

Comment:
```suggestion
Show extras, sometimes called specials, can be added in the `Season 00` folder. If supported by your metadata provider those files will be matched. In case your metadata provider does not provide information about the extra, it is recommended to use a name which describes the content of the special instead of naming it `Episode S00Exy.mkv`. This is done to avoid wrong metadata being pulled for the extra and to provide a proper presentation.
```

May need to be reworded since I just did this while reading through.

Reply:
> May need to be reworded since I just did this while reading through.

Just checked it and from my point of view, these suggestions look fine the way they are. ;)